tests/common/integration_features/test_custom_policies_integration.py

```diff
@@ -242,26 +250,111 @@ def test_policy_load(self):
         self.assertEqual(len([r for r in report.failed_checks if r.check_id == 'mikepolicies_AWS_1625063607541']), 0)
 
-        report = tf_runner.run(root_folder=test_files_dir, runner_filter=RunnerFilter(skip_checks=['mikepolicies_aws_1625063842021']))
+        report = tf_runner.run(root_folder=test_files_dir,
+                               runner_filter=RunnerFilter(skip_checks=['mikepolicies_aws_1625063842021']))
         self.assertEqual(len([r for r in report.failed_checks if r.check_id == 'mikepolicies_aws_1625063842021']), 0)
         self.assertEqual(len([r for r in report.failed_checks if r.check_id == 'mikepolicies_AWS_1625063607541']), 1)
 
-        report = cfn_runner.run(root_folder=test_files_dir, runner_filter=RunnerFilter(checks=['kpande_AWS_1635187541652']))
+        report = cfn_runner.run(root_folder=test_files_dir,
+                                runner_filter=RunnerFilter(checks=['kpande_AWS_1635187541652']))
         self.assertEqual(len([r for r in report.failed_checks if r.check_id == 'kpande_AWS_1635187541652']), 6)
         self.assertEqual(len([r for r in report.failed_checks if r.check_id == 'kpande_AWS_1635180094606']), 0)
 
-        report = cfn_runner.run(root_folder=test_files_dir, runner_filter=RunnerFilter(checks=['kpande_AWS_1635180094606']))
+        report = cfn_runner.run(root_folder=test_files_dir,
+                                runner_filter=RunnerFilter(checks=['kpande_AWS_1635180094606']))
         self.assertEqual(len([r for r in report.failed_checks if r.check_id == 'kpande_AWS_1635180094606']), 1)
         self.assertEqual(len([r for r in report.failed_checks if r.check_id == 'kpande_AWS_1635187541652']), 0)
 
-        report = cfn_runner.run(root_folder=test_files_dir, runner_filter=RunnerFilter(skip_checks=['kpande_AWS_1635180094606']))
+        report = cfn_runner.run(root_folder=test_files_dir,
+                                runner_filter=RunnerFilter(skip_checks=['kpande_AWS_1635180094606']))
         self.assertEqual(len([r for r in report.failed_checks if r.check_id == 'kpande_AWS_1635180094606']), 0)
         self.assertEqual(len([r for r in report.failed_checks if r.check_id == 'kpande_AWS_1635187541652']), 6)
 
+    def test_pre_scan_with_cloned_checks(self):
+        instance = BcPlatformIntegration()
+        instance.skip_policy_download = False
+        instance.platform_integration_configured = True
+        custom_policies_integration = CustomPoliciesIntegration(instance)
+
+        # mock _get_policies_from_platform method
+        custom_policies_integration._get_policies_from_platform = types.MethodType(_get_policies_from_platform,
+                                                                                   custom_policies_integration)
+
+        custom_policies_integration.pre_scan()
+        self.assertEqual(1, len(custom_policies_integration.policies))
+        self.assertEqual(1, len(custom_policies_integration.bc_cloned_checks))
+
```

Comment:
```python
from checkov.common.checks_infra.registry import Registry, get_graph_checks_registry  # adjust the import in th top

        ...

        get_graph_checks_registry("cloudformation").checks = []. # this you don't actually need, because there are no CFN checks, but better to be safe than sorry
        get_graph_checks_registry("terraform").checks = []
```

ad this also at the bottom of the test under this one